sean/joke.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+  <head>
+      <meta charset="utf-8">
+      <title>Ew, David</title>
+      <!-- <link rel="preconnect" href="https://fonts.gstatic.com"> -->
+      <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;500&display=swap" rel="stylesheet">
```

Comment:
Since you aren't using the `font-weight` property in your CSS, you can remove `300` from the list of weights. 

```suggestion
      <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@500&display=swap" rel="stylesheet">
```

The benefit of this is that there is less data your user has to download when using your site, since each font-weight is it's own complete font. So this could be a `performance` improvement!